scripts/chroot_reconst.sh

```diff
@@ -64,6 +64,9 @@ apt-get -qq -y remove xscreensaver
 # This switch offs the sleep mode
 sudo systemctl mask sleep.target suspend.target hibernate.target hybrid-sleep.target
 
+# This switch offs system sounds
+sed -i 's\# set bell-style none\set bell-style none\g' /etc/inputrc
```

Comment:
See also https://www.linuxquestions.org/questions/ubuntu-63/set-bell-style-none-doesn%27t-silence-my-bell-316731/

Reply:
Technically I would prefer you to make a dedicated script file with a title like scripts/packages/sound_off.sh or similar where all these necessary commands for the feature are.

Reply:
@tabesin I will seen the article and will do the required changes,
also lets move the all the commands into the specific files also, this will help in executing commands when sent from the generator

Reply:
Could be also a trick to avoid if statements:
You exec feature.sh; where i389 and amd64 differ you execute feature-$arch.sh with two files existing.

Personally I think we should add at least add dummy files per unimplemented feature. That makes it easier to keep things under control.